### PR TITLE
Re-enable curl testcases for ssl suite

### PR DESCRIPTION
**Pull request by ckulal**
# Description

Re-enable curl testcases for ssl suite with the support provided in ceph-qe-scripts:
https://github.com/red-hat-storage/ceph-qe-scripts/pull/617

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/rgw/tier-1_rgw_upgrade_ssl_71GA_to_71x_latest.yaml [hunk 1]
```diff
@@ -181,15 +181,15 @@ tests:
...
-  #       timeout: 500
+  - test:
+      name: Test rgw through CURL
+      desc: Test rgw through CURL
+      polarion-id: CEPH-83575572
+      module: sanity_rgw.py
+      config:
+        script-name: ../curl/test_rgw_using_curl.py
+        config-file-name: ../../curl/configs/test_rgw_using_curl.yaml
+        timeout: 500
```
Comment (psathyan) [inline suites/reef/rgw/tier-1_rgw_upgrade_ssl_71GA_to_71x_latest.yaml:192]: nit: AFAIK, `sanity_rgw.py` does not honor the timeout. Please be mindful of the configuration.

Comment (ckulal) [reply suites/reef/rgw/tier-1_rgw_upgrade_ssl_71GA_to_71x_latest.yaml]: For now I have removed timeout from testcase level,
we do have a plan to to set timeout from sanity_rgw.py file, so testcase will take timeout from module itself.
note: this particular testcase will not take time more than 1 hr

**Review by psathyan — COMMENTED** [review]
/lgtm

**Review by viduship — APPROVED** [review]

**Review by anrao19 — APPROVED** [review]
LGTM

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**